main.js

```diff
@@ -0,0 +1,73 @@
+      // part 1
+      
+       snakewaterMontana = {
+      paleontologist: "Dr. Alan Grant",
+      depth: "10 meters",
+       specimen: "Velociraptor"
+       }
+
+     var guestOfHonor = snakewaterMontana.paleontologist;
+
+     var cleverGirl = snakewaterMontana.specimen; 
+      console.log(cleverGirl);
+  
+
+// part2
+     nicaragua = {
+    depth: "200 meters",
+    annualBudget: 1500000,
+    specimens: [
+      "Tyrannosaurus Rex",
+      "Stegosaurous",
+      "Triceratops",
+      "Velociraptor"
+    ]
+  }
+ 
+
+   var nicaraguanSpecimens = 
+    
+  ["Tyrannosaurus Rex",
+  "Stegosaurous",
+  "Triceratops",
+  "Velociraptor"]
```

Comment:
what is the point of adding the array manually can you imagine adding an array with 100 elements!! do you have to write them all ? all you need is to access the array located inside the object then create a new variable to assign the value `var nicaraguanSpecimens = nicaragua.specimens;`